load-tester/load-tests/broadcasts/config/index.js

```diff
@@ -0,0 +1,17 @@
+const cliOptions = require('./cli-options');
+
+const config = {
+  cliOptions,
+};
+
+const broadcastSettingsBaseUrl = process.env.DS_CONVERSATIONS_BROADCASTS_SETTINGS_URL || 'http://puppet:totallysecret@localhost:5100/api/v1/broadcast-settings';
+const broadcastId = process.env.DS_CONVERSATIONS_BROADCAST_SETTINGS_ID || 'tacosfest';
```

Comment:
I was confused when setting this up what a `SETTINGS_ID` was, especially because the `.env.example` doesn't contain a valid one.

What do you think about just changing it to `DS_CONVERSATION_BROADCAST_ID`?

It'd be nice to get https://github.com/DoSomething/gambit-conversations/issues/116 before we launch to avoid using the `broadcastId` field at all, and just use the Contentful entry ID.

Reply:
I'll change the variable name to `DS_CONVERSATIONS_BROADCAST_ID `. In the context of this script and the route we are GETting, even if #116 makes it before launch, I think calling it broadcastId is valid. I agree getting to #116 would be sweet, there might be an opportunity to refactor that as we add more testing coverage in the coming weeks prior to shipping it 📦 🚀.